src/scripts.ts

```diff
@@ -0,0 +1,22 @@
+export const projectScripts = () => {
```

Comment:
Will we ever run into a situation where the Builder is returning project scripts that aren't compatible with the project's SDK version?

Reply:
@mwbrooks I may be misunderstanding your question, but since the SDK (builder) is controlling the output returned in this method, it should always be compatible with itself?

Reply:
@filmaj I'm probably just confusing myself 😅 

For example considering the `run` script: will the builder will be able to ensure that the code written - regardless of the SDK version - will work for with `deno-slack-runtime`? It seems like it will, but I wanted to ask regardless.

Reply:
I also see validity to the same question about whether or not `deno-slack-sdk` will remain compatible with the version of the `deno-slack-builder` specified by the `get-hooks` command

Reply:
It's a good point that I did not fully understand initially 😄 
I am not so much worried about builder and runtime being incompatible with each other, as they have separate areas of responsibility. However, @shapirone's example I think better illustrates the potential for a problem: if the `deno-slack-sdk` makes some major change (perhaps changing the directory name where function code exists), that would necessitate updating both builder and runtime components to work with the change as well.

If anything, to me this is a responsibility that us as Slack developers need to keep in mind: how the various Hermes projects interact and work together, and we need to be vigilant about changes that could introduce incompatibilities with the other projects.